### PR TITLE
ISSD-3199 [PRM] MDF Activity Expired inside MDF Claim form

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx [hunk 1]
```diff
@@ -73,6 +74,22 @@ const MDFClaimPage = ({
 		}
 	).length;
 
+	const displayActivityClaim = (activity: MDFClaimActivity) => {
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx:77]:
```suggestion
	const isDisplayableMDFActivityClaim = (activity: MDFClaimActivity) => {
```

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx [hunk 2]
```diff
@@ -73,6 +74,22 @@ const MDFClaimPage = ({
...
+		const claimableActivityByStatus =
+			activity.activityStatus?.key !== Status.EXPIRED.key &&
+			activity.activityStatus?.key !== Status.CANCELED.key &&
+			!activity.claimed;
+
+		const editableClaimActivityByStatus =
+			!!activity.id && !activity.selected;
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx:84]:
```suggestion
			Boolean(activity.id) && !activity.selected;
```

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx [hunk 3]
```diff
@@ -73,6 +74,22 @@ const MDFClaimPage = ({
...
+
+		const displayActivityClaim = activity.id
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx:86]:
```suggestion
		const isDisplayable = activity.id
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx (hunk 1 above)
Comment (abfranca) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx]: Done!

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx (hunk 2 above)
Comment (abfranca) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx]: Done!

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx (hunk 3 above)
Comment (abfranca) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx]: Done!

**Comment by abfranca** [comment]
ci:test:sf

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**